src/Resources/views/storefront/component/product/card/action.html.twig

```diff
@@ -0,0 +1,53 @@
+{% sw_extends '@Storefront/storefront/component/product/card/action.html.twig' %}
+
+{% block component_product_box_action_inner %}
```

Comment:
I think you should not overwrite the whole `{% block component_product_box_action_inner %}` block
Consider the following code
```
{% sw_extends '@Storefront/storefront/component/product/card/action.html.twig' %}

{% block component_product_box_action_form %}

                {% block component_product_box_action_buy_redirect_input %}
                    <input type="hidden"
                           name="redirectTo"
                           value="frontend.cart.offcanvas"/>
                {% endblock %}

                {% block page_product_detail_buy_product_buy_info %}
                    <input type="hidden"
                           name="lineItems[{{ id }}][id]"
                           value="{{ id }}">
                    <input type="hidden"
                           name="lineItems[{{ id }}][referencedId]"
                           value="{{ id }}">
                    <input type="hidden"
                           name="lineItems[{{ id }}][type]"
                           value="product">
                    <input type="hidden"
                           name="lineItems[{{ id }}][stackable]"
                           value="1">
                    <input type="hidden"
                           name="lineItems[{{ id }}][removable]"
                           value="1">
                    <input type="hidden"
                           name="lineItems[{{ id }}][quantity]"
                           value="1">
                {% endblock %}

                <button class="btn btn-block btn-buy"
                        disabled="disabled"
                        title="{{ "listing.boxAddProduct"|trans|striptags }}">
                    {{ "listing.boxAddProduct"|trans|sw_sanitize }}
                </button>
  {% endblock %}

```